DEVELOPMENT.rst

```diff
@@ -0,0 +1,52 @@
+Developing on edx-sga
+=====================
+
+Setup (including devstack setup)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. `Install vagrant <http://docs.vagrantup.com/v2/installation/>`_.
+#. `Install virtualbox <https://www.virtualbox.org/wiki/Downloads>`_.
+#. Set up devstack::
+
+    mkdir devstack
+    cd devstack
+    curl -L https://raw.githubusercontent.com/edx/configuration/master/vagrant/release/devstack/Vagrantfile > Vagrantfile
+    vagrant plugin install vagrant-vbguest
+    vagrant up
+    cd themes/
+
+#. Fork https://github.com/mitodl/edx-sga.git to your own github account.
+#. Set up your development environment::
+
+    git clone https://github.com/your-name/edx-sga.git    
+    vagrant ssh    
+    sudo su edxapp    
+    cd /edx/app/edxapp/themes/    
```

Comment:
If you have sudo'ed to become the ``edxapp`` user, the themes directory is ``./themes``.